pkg/services/sqldb/types.go

```diff
@@ -2,13 +2,20 @@ package sqldb
 
 import "github.com/Azure/open-service-broker-azure/pkg/service"
 
-// ProvisioningParameters encapsulates MSSQL-specific provisioning options
-type ProvisioningParameters struct {
-	ServerName      string `json:"server"`
+// ServerProvisioningParams encapsulates MSSQL-server specific provisioning
+//options
+type ServerProvisioningParams struct {
 	FirewallIPStart string `json:"firewallStartIPAddress"`
 	FirewallIPEnd   string `json:"firewallEndIPAddress"`
 }
 
+// DBProvisioningParams encapsulates MSSQL-specific provisioning options
+type DBProvisioningParams struct {
+}
+
+// instance details are common to the DB only and All In One Service.
```

Comment:
I think there are a few attributes here that are, perhaps, not relevant to db-only. wdyt?